plugins/heroku/src/setSlug.ts

```diff
@@ -30,6 +32,10 @@ function setSlug(logger: Logger, slug: string): Promise<void[]> {
       })
       .then((response) => gtg(logger, response.app.name, 'production', false))
   )
+  
+  for(const id of appIds) {
+    await setConfigVars(logger, id, 'production', systemCode)
```

Comment:
**question**: why do we set the vault vars in the `setSlug` function? are the two actions related?

Reply:
you're right, the naming isn't very helpful - I put it in setSlug because setSlug is only called in heroku/src/tasks/production.ts, and setSlug is the place where we get the appIds.

I'm thinking to rename 'setSlug' to 'promoteStagingToProduction'. what are your thoughts?

Reply:
I'd say it would be better to split it out into a different function and either call both this and `setSlug` from a `promoteStagingToProduction` function or just directly from the production task. I expect that we'll want to add more logic setting the environment variables anyway